### PR TITLE
*: Cleanup UseNewPlanner

### PR DESCRIPTION
**Pull request by shenli**
@hanfei1991 @XuHuaiyu @coocood PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM, We should remove the definition of UseNewPlanner

File: plan/logical_plan_builder.go [hunk 1]
```diff
@@ -25,9 +25,6 @@ import (
 	"github.com/pingcap/tidb/util/types"
 )
 
-// UseNewPlanner means if use the new planner.
-var UseNewPlanner = true
```
Comment (shenli) [inline plan/logical_plan_builder.go]: @ngaut It is deleted.

Comment (ngaut) [inline plan/logical_plan_builder.go]: Gotcha.

**Comment by hanfei1991** [comment]
LGTM!

**Pull request merged**